sympy/solvers/tests/test_ode.py

```diff
@@ -34,78 +34,47 @@ def test_linear_2eq_order1():
     eq1 = (Eq(diff(x(t),t), 9*y(t)), Eq(diff(y(t),t), 12*x(t)))
     sol1 = [Eq(x(t), 9*C1*exp(6*sqrt(3)*t) + 9*C2*exp(-6*sqrt(3)*t)), \
     Eq(y(t), 6*sqrt(3)*C1*exp(6*sqrt(3)*t) - 6*sqrt(3)*C2*exp(-6*sqrt(3)*t))]
-    s = dsolve(eq1)
-    assert s == sol1
-    s = [(l.lhs, l.rhs) for l in s]
-    assert eq1[0].subs(s).doit().simplify()
-    assert eq1[1].subs(s).doit().simplify()
+    assert checksysodesol(eq1, sol1) == (True, [0, 0])
 
     eq2 = (Eq(diff(x(t),t), 2*x(t) + 4*y(t)), Eq(diff(y(t),t), 12*x(t) + 41*y(t)))
     sol2 = [Eq(x(t), 4*C1*exp(t*(sqrt(1713)/2 + 43/2)) + 4*C2*exp(t*(-sqrt(1713)/2 + 43/2))), \
```

Comment:
`S(43)/2` instead of `43/2` (Similar change as suggested [here](https://github.com/sympy/sympy/pull/15133#discussion_r212488513))